scripts/build-integration.sh

```diff
@@ -4,6 +4,6 @@ set -eox pipefail
 echo ">> Building contract"
 
 rustup target add wasm32-unknown-unknown
-cargo build -p sweat_booster --target wasm32-unknown-unknown --profile=contract --features integration-test,
+cargo build -p sweat-booster --target wasm32-unknown-unknown --profile=release --features integration-test,
 
-cp ./target/wasm32-unknown-unknown/contract/sweat_booster.wasm res/sweat_booster.wasm
+cp ./target/wasm32-unknown-unknown/release/sweat_booster.wasm res/sweat_booster.wasm
```

Comment:
Why release?

Reply:
Why not? It's just a default profile from `cargo near` template

Reply:
Because we deploy binaries built with `contract` profile. I think it would be better to to test with this profile too.